Sources_v3/Database/DatabaseSession.swift

```diff
@@ -29,6 +29,17 @@ protocol CurrentUserDatabaseSession {
     /// If there is no current user, the error will be thown
     func saveCurrentUserUnreadCount(count: UnreadCount) throws
     
+    /// Updates the `CurrentUserDTO.devices` with the provided `DevicesPayload`
+    /// If there's no current user set, an error will be thrown.
+    func saveCurrentUserDevice(_ device: DevicePayload) throws
+    
+    /// Updates the `CurrentUserDTO.devices` with the provided `DevicesPayload`
+    /// If there's no current user set, an error will be thrown.
+    func saveCurrentUserDevices(payload: DevicesPayload) throws
+    
```

Comment:
These two functions probably don't have to a requirement of the protocol? If you really want to provide two overloads for this (I don't think it's necessary), I'd suggest making only one specified by the protocol, and define the other in the extension.

Reply:
Converted it to one function in 107624943165ad39afd991a7fd9aba8d58c1a28a